### ISSUE TITLE
Dropdown showArrow should default to false

### ISSUE THREAD

**Issue by pmsorhaindo** [BrandwatchLtd/axiom#590]
Breaking change enhancement:

Too minor to warrant a breaking change in and of itself, but the next axiom breaking change should include this to prevent the need of `<Dropdown showArrow={ false }>`. If people feel otherwise feel free to debate and close this issue if the consensus is contrary.

**Comment by HHogg** [comment]
I'm in favour of this. I'm not a big fan of component APIs defaulting to adding behaviour. `withArrow` would be nicer than writing `showArrow={ false }`.

**Comment by lpoulter** [comment]
Makes sense to me. In the front end i've only ever seen it used with `showArrow={ false }`
